workitem/workitem_repository.go

```diff
@@ -687,39 +690,58 @@ func (r *GormWorkItemRepository) Fetch(ctx context.Context, spaceID uuid.UUID, c
 // 		WHERE (iterations.space_id = '33406de1-25f1-4969-bcec-88f29d0a7de3'
 // 		and work_items.deleted_at IS NULL) GROUP BY IterationId
 func (r *GormWorkItemRepository) GetCountsPerIteration(ctx context.Context, spaceID uuid.UUID) (map[string]WICountsPerIteration, error) {
-	var res []WICountsPerIteration
-	db := r.db.Table("work_items").Select(`iterations.id as IterationId, count(*) as Total,
-				count( case fields->>'system.state' when 'closed' then '1' else null end ) as Closed`).Joins(`left join iterations
-				on fields@> concat('{"system.iteration": "', iterations.id, '"}')::jsonb`).Where(`iterations.space_id = ?
-				and work_items.deleted_at IS NULL`, spaceID).Group(`IterationId`).Scan(&res)
+	var itrs []*iteration.Iteration
+	db := r.db.Debug()
+	db = db.Table("iterations").Select("*").Where("space_id = ? AND deleted_at IS NULL", spaceID).Scan(&itrs)
```

Comment:
And why not just do this?

```go
db = db.Model(&iteration.Iteration{}).Where("space_id = ?", extendedTypeID).Scan(&itrs)
```

The benefit is that you don't need to hard-code the table name `"iterations"` into the query. It will be either guessed by looking at the Model name or taken from a piece of code like this:

```go
func (i *Iteration) TableName() string {
	return "iterations"
}
```